Makefile

```diff
@@ -31,7 +31,8 @@ install: install-scripts install-conf install-systemd
 # target: install-scripts - Install executables.
 install-scripts:
 	install -d $(DEST_SCRIPTS)
-	install -m 0744 $(SRCS_SCRIPTS) $(DEST_SCRIPTS)
+	install -m 0744 $(filter-out %/resticw, $(SRCS_SCRIPTS)) $(DEST_SCRIPTS)
+	install -m 0755 usr/local/sbin/resticw $(DEST_SCRIPTS)
```

Comment:
Using 755 here as, the user should be able to find `resticw` discoverable, but by using 744, `which resticw` shows nothing, as well as tab  auto-completion does not help when typing the command.

Reply:
:+1: